tests/unit/timesheet/api/test_task_viewset.py

```diff
@@ -208,6 +208,13 @@ def _as_quantized_decimal(n: int | float | Decimal) -> str:
                     'task': None,
                 },
             ],
+            'days': {'2024-01-01': {'hol': True, 'nwd': True},
```

Comment:
https://taiga.singlewave.co.uk/project/krm3/task/90
days should be a list, not a dictionary